web/packages/teleterm/src/agentCleanupDaemon/agentCleanupDaemon.js

```diff
@@ -59,13 +68,36 @@ if (!process.send) {
   );
 }
 
-const logLine = `parent=${parentPid} agent=${agentPid} self=${process.pid} cluster=${rootClusterUri}`;
+const logger = createLogger({
+  level: 'info',
+  exitOnError: false,
+  format: format.combine(
+    format.timestamp({
+      format: 'DD-MM-YY HH:mm:ss',
+    }),
+    format.simple()
+  ),
+  transports: [
+    new transports.Console(),
+    new transports.File({
+      maxsize: 4194304, // 4 MB - max size of a single file
+      maxFiles: 5,
+      dirname: logsDir,
+      filename: `cleanup.log`,
+    }),
```

Comment:
From what I've seen, by default `fork` is going to pass the equivalent of `stdio: 'inherit'` to `spawn`. This will make the cleanup daemon logs appear in the terminal in dev mode, although the logs will not be colorized like all the other.

Using `pipeProcessOutputIntoLogger` here would require some extra options just for this use case.

Overall, implementing this made me realize that the approach we took with the main process orchestrating the logs of child processes is wrong. Each child should manage its own logs and what the main process should do at most is doing the equivalent of `tail -F` on them in dev mode. But that's a topic for another discussion.